src/pks/surface_balance/constitutive_relations/litter/micropore_macropore_flux_evaluator.cc

```diff
@@ -37,10 +88,12 @@ void
 MicroporeMacroporeFluxEvaluator::InitializeFromPlist_()
 {
   // Set up my dependencies
-  Key micro_domain = Keys::getDomain(my_keys_.front().first);
-  Key macro_domain = Keys::readDomainHint(plist_, micro_domain, "micropore", "macropore");
-  auto tag = my_keys_.front().second;
 
+  //Key micro_domain = Keys::getDomain(my_key_);
+  Key micro_domain = plist_.get<std::string>("micropore domain name", "domain");
+  Key macro_domain = plist_.get<std::string>("macropore domain name", "macropore");
```

Comment:
Is there a reason these had to change?  I think this should still be valid.  One of the domains is given by the name of this variable.  The other can be guessed.